code-studio/test/audio/audio_test.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html dir='ltr'>
+  <head>
+    <script>
+      // fake module for sounds
+      window.module = {};
+    </script>
+    <script src="/src/js/Sounds.js"></script>
+    <script>
+      var Sounds = module.exports;
+      var sounds = new Sounds();
+      sounds.register({
+        id: 'intro',
+        ogg: 'assets/vignette4-intro.ogg',
+        mp3: 'assets/vignette4-intro.mp3'
```

Comment:
These used to be served by dashboard. Now we have symlinks in code-studio to the copies of the assets in apps, and server via http-server.